### ISSUE TITLE
MSBuild.NSwag fails on .NET 8 when simplified output paths enabled

### ISSUE THREAD

**Issue by jasontaylordev** [RicoSuter/NSwag#4485]
The upcoming release of .NET 8 introduces [simplified output paths](https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8#simplified-output-paths), an option to simplify the output path and folder structure for build outputs (e.g. bin and obj). **NSwag.MSBuild** fails when this option is enabled. 

The repository [DotNet8NswagIssue](https://github.com/jasontaylordev/DotNet8NswagIssue) was created to demonstrate the issue so that it can be resolved. The repository includes a README that details steps to reproduce and steps to workaround this issue.

**Comment by GraemeWellington** [comment]
Hi
I am not sure if this resolved but here is my storey.
I just revisited one of my clean architecture solutions which was sitting at .net 7 and I had implemented NSwag functionality following Jason Taylors videos including the mods to .csproj for the Post Build process. After upgrdaing to .net 8 I ran into the same problem as this post. I then worked through a few options and it now seems to work without the work around to the Directory.Build.props file.

After the minor mods to get the app running was to add the argument 'config.nswag' to the <Exec line in .csproj (I note this was in some of Rico's code in other posts) - here are my changes:

[DotNet8NswagIssue-NET8-Fixes.txt](https://github.com/RicoSuter/NSwag/files/14155784/DotNet8NswagIssue-NET8-Fixes.txt)
